src/components/MoneyReportHeader.js

```diff
@@ -69,16 +69,19 @@ function MoneyReportHeader({session, personalDetails, policy, chatReport, report
     const isPolicyAdmin = policyType !== CONST.POLICY.TYPE.PERSONAL && lodashGet(policy, 'role') === CONST.POLICY.ROLE.ADMIN;
     const isManager = ReportUtils.isMoneyRequestReport(moneyRequestReport) && lodashGet(session, 'accountID', null) === moneyRequestReport.managerID;
     const isPayer = policyType === CONST.POLICY.TYPE.CORPORATE ? isPolicyAdmin && isApproved : isPolicyAdmin || (ReportUtils.isMoneyRequestReport(moneyRequestReport) && isManager);
+    const isDraft = ReportUtils.isReportDraft(moneyRequestReport);
     const shouldShowSettlementButton = useMemo(
-        () => isPayer && !isSettled && !moneyRequestReport.isWaitingOnBankAccount && reportTotal !== 0 && !ReportUtils.isArchivedRoom(chatReport),
-        [isPayer, isSettled, moneyRequestReport, reportTotal, chatReport],
+        () => isPayer && !isDraft && !isSettled && !moneyRequestReport.isWaitingOnBankAccount && reportTotal !== 0 && !ReportUtils.isArchivedRoom(chatReport),
+        [isPayer, isDraft, isSettled, moneyRequestReport, reportTotal, chatReport],
     );
     const shouldShowApproveButton = useMemo(() => {
         if (policyType !== CONST.POLICY.TYPE.CORPORATE) {
             return false;
         }
-        return isManager && !isApproved && !isSettled;
-    }, [policyType, isManager, isApproved, isSettled]);
+        return isManager && !isDraft && !isApproved && !isSettled;
+    }, [policyType, isManager, isDraft, isApproved, isSettled]);
+    const shouldShowSubmitButton = isDraft;
```

Comment:
Admin can submit and approve it is own report when `Prevent self approval` is enabled. So we should have prevented submitting it, ref: https://github.com/Expensify/App/issues/36425